calico/felix/test/test_config.py

```diff
@@ -76,6 +76,10 @@ def test_default_config(self):
 
             # Test defaulting.
             self.assertEqual(config.ETCD_ADDR, "localhost:4001")
+            self.assertEqual(config.ETCD_SCHEME, "http")
+            self.assertEqual(config.ETCD_KEY_FILE, "")
```

Comment:
I think elsewhere we allow the user to use the string "none" in the config file and we translate that to Python's `None` to pass around.

We should be consistent with that, I think.